source/includes/_payments.md

```diff
@@ -463,16 +463,21 @@ Authorization: Basic ...
 HTTP/1.1 202 OK
 Content-Type: application/json
 
-""
+{
+  "id" : "walletId"
```

Comment:
Do we want this to be a more concrete example?  Worth having an actual UUID here to set expectations?  Might be useful in terms of platforms thinking about  column widths?

Reply:
Good shout!